### PR TITLE
Add educational content to style guide

### PR DESCRIPTION
**Pull request by faase**
#### Summary

To keep readers engaged with educational content the style guide needed to allow some new stylistic devices.

Also removed some newlines.

### CODE REVIEW

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-same section. In most cases, a document has a title, an introductory paragraph, and one or more sections.
+Structure and organization are an important part of a document's ease of use and its understandability. Information should be organized and presented in a logical order, with similar subjects grouped together in the same section. In most cases, a document has a title, an introductory paragraph, and one or more sections.
 
-Try to keep only one topic in a page. Shorter topics are easier to reuse in other documents, are 
-easier to write and edit, and are easier to translate.
+Try to keep only one topic in a page. Shorter topics are easier to reuse in other documents, are easier to write and edit, and are easier to translate.
+
+### Educational guides
+
+An exception are educational guides. In their abstract they set clear expectations for a well defined audience, can have more then one topic in a page, and do not require an introductory paragraph. First person writing can be used, as they need to keep the reader interested in the subject.
```
Comment (jasonblais) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: Let's make the exceptions clear, and add a concise note on why those exceptions are made for educational guides.

E.g. later in this page we have a note about the abstract

```
The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the description is displayed in the search results along with the page title. People read the description to help them decide if the document is the one that they want.
```

I'm wondering how much educational guides deviate from this?

**Review by jasonblais — CHANGES_REQUESTED** [review]

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 1 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: The Idea was to write a single paragraph telling before hand any changes to the below sections for someone to write educational content, so they only have to read a single paragraph if they are already  familiar with the style/voice/tone guidelines.

Of course the deviations can be mentioned per paragraph, but someone already familiar with the document would need to reread everything in order to get all differences.

**Comment by faase** [comment]
It has been reworked according to our earlier conversation and (hopefully) all conflicts have been resolved.

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 2]
```diff
@@ -23,38 +17,32 @@ The title appears in the left-hand navigation and at the top of the table of con
...
-navigate the document. The table of contents should be added below the abstract. 
+If the document contains more than four sections or subsections, add a table of contents to help the user navigate the document. The table of contents should be added below the abstract. 
 
 ### Introductory paragraph
 
-Each page should have an introduction that acts as a short description of the document. The short description
-should be a single paragraph of no more than three sentences. 
+Each page should have an introduction that acts as a short description of the document. The short description should be a single paragraph of no more than three sentences. 
+
+An introductory paragraph is not needed for educational guides.
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: Question: In the [deployment guide](https://docs.mattermost.com/deployment/mobile-app-deployment.html) there's an "About this Guide" section with learning objectives. Would that be considered the CE equivalent of an introduction?

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md [hunk 3]
```diff
@@ -77,6 +77,8 @@ Use the second person and avoid the first person.
 
 For example: "View the status in the Status pane" as opposed to "We’ll view the status in the Status pane".
 
+An exception are educational guides, as first person writing helps them to keep the reader interested in  the subject.
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]:
```suggestion
An exception is Customer Education content which is written in first-person to align more closely with the subject matter.
```
I'm 2/5 on this exact wording but I'm inclined toward either a more substantial reason or no reason at all.

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 4]
```diff
@@ -23,38 +17,32 @@ The title appears in the left-hand navigation and at the top of the table of con
 
 ### Document sections
 
-To make pages easier for people to quickly scan for the content that they're looking for, break your 
-document up into logical sections. Each section should have a heading, and the heading should relate 
-to the content of the section. A section heading is not required if you have only one section.
+To make pages easier for people to quickly scan for the content that they're looking for, break your document up into logical sections. Each section should have a heading, and the heading should relate to the content of the section. A section heading is not required if you have only one section.
 
 **Use sentence case for the section heading (e.g. "This section is about types of headings") except where a word/phrase is a proper noun (e.g. "Mattermost"), the name of a country/region (e.g. "The United States of America"), and so on.**
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: There's an extra line here that I think we can remove.

**Review by jasonblais — DISMISSED** [review]
No other feedback than what Justine shared above, thanks!

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 2 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: I considered it part of the abstract as the Learning Objectives summarize the document.

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md (hunk 3 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: I love your wording! To me keeping the reader engaged is the most important part of a CE guide. If I want to educate someone on a matter by using a text, I first of all have to make sure they read it.

How about a mix like 

```suggestion
An exception is Customer Education content which can be written in first-person to keep the audience focused and interested.
```

?

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 5]
```diff
@@ -23,38 +17,32 @@ The title appears in the left-hand navigation and at the top of the table of con
...
 
 ### Abstract
 
-The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. 
-It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the 
-description is displayed in the search results along with the page title. People read the description to 
-help them decide if the document is the one that they want.
+The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the description is displayed in the search results along with the page title. People read the description to help them decide if the document is the one that they want. 
+
+For educational guides they additionally set clear expectations for a well defined audience.
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]:
```suggestion
An exception is Customer Education content where this is replaced by Learning Objectives which set clear expectations for a well-defined audience.
```

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 2 above)
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]:
```suggestion
An exception is Customer Education content where the introduction is replaced by the Learning Objectives section.
```

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md (hunk 3 above)
Comment (justinegeffen) [reply operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: I understand that. My only change is:
"An exception is Customer Education content which is written in first-person to keep the audience focused and interested."

As an aside, I am wary of stating that first-person keeps the audience focused and interested as first-person is more of a tool used in fiction writing than technical content. That said, my perspective is biased and comes from the opposite end of the spectrum and so is just an opinion/comment (especially as I don't know much about sales/marketing writing). :)

Comment (faase) [reply operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: Wouldn't that make it impossible to write educational content from second person?

Comment (justinegeffen) [reply operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: I didn't realize that was an option, so in that case "can be" would be more appropriate. Perhaps we can say "can be written in first or second person"?

**Review by justinegeffen — DISMISSED** [review]

**Comment by jasonblais** [comment]
@faase Can you help confirm these are the changes we're adding? 

> [abstract] An exception is Customer Education content where this is replaced by Learning Objectives which set clear expectations for a well-defined audience.

> [intro paragraph] An exception is Customer Education content where the introduction is replaced by the Learning Objectives section.

> [person] An exception is Customer Education content which can be written in first or second person to keep the audience focused and interested.

**Comment by faase** [comment]
@jasonblais Yes, these are the contextual changes, there is one more change in 
[Documentation Style Guide] It must be accurate and clear, be presented with a style and tone that is appropriate for technical **or educational content**, and easy to navigate.

> @faase Can you help confirm these are the changes we're adding?
> 
> > [abstract] An exception is Customer Education content where this is replaced by Learning Objectives which set clear expectations for a well-defined audience.
> 
> > [intro paragraph] An exception is Customer Education content where the introduction is replaced by the Learning Objectives section.
> 
> > [person] An exception is Customer Education content which can be written in first or second person to keep the audience focused and interested.

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 6]
```diff
@@ -23,38 +17,31 @@ The title appears in the left-hand navigation and at the top of the table of con
...
-
 ### Abstract
 
-The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. 
-It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the 
-description is displayed in the search results along with the page title. People read the description to 
-help them decide if the document is the one that they want.
+The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the description is displayed in the search results along with the page title. People read the description to help them decide if the document is the one that they want. 
+
+An exception is Customer Education content where this is replaced by Learning Objectives which set clear expectations for a well-defined audience.
```
Comment (jasonblais) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: Out of curiosity - if learning objectives set 'clear expectations for a well-defined audience', how much will it differ from the abstract which provides 'a high-level overview of the content .. and outlines the value or goal to the reader', and which 'people read the description to help them decide if the document is the one that they want'

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md [hunk 7]
```diff
@@ -77,6 +77,8 @@ Use the second person and avoid the first person.
 
 For example: "View the status in the Status pane" as opposed to "We’ll view the status in the Status pane".
 
+An exception is Customer Education content which can be written in first or second person to keep the audience focused and interested.
```
Comment (jasonblais) [inline operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: Propose we include an example. It is not clear to a new reader why using second person doesn't keep the audience focused the same way as using first person.

**Review by jasonblais — COMMENTED** [review]
I don't mean to prolong the review process, but I'm reading the updates with fresh eyes and have some questions.

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 6 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: Very good find @jasonblais , here are my thoughts:

### Educational Guides - this PR

> clear expectations
- How the reader benefits from reading the guide - what  exact knowledge is taught
- Other expectations
    - How long working through/with the guide will take 
    - What similar topics are not covered
    - The audience is at the center of attention (what can the reader expect)

> for a well-defined audience
 - This audience can be everything from very narrow (System Administrators) to a larger audience (System Administrators, Project Managers, and IT Leaders in general), but it has to be named

### Existing Documentation

> a high-level overview of the content .. and outlines the value or goal to the reader
- What topics the document touches
- *outlines the value or goal* is very universal and unspecific
- the document itself is the center of attention (what does the document bring to the reader)
- reader is not specified in the document itself but by docs category (User, Administrator, Developer)

### TL;DR

- Educational guides have no need to outline the content but set more specific expectations
- different focus in language, focus is the reader as opposed to focus is the document

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md (hunk 7 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: Here's an example from the *Mobile App Deployment Guide*

>A mobile deployment is not a small project. While Mattermost provides options to make it as easy as possible, it will take time and iteration.
>
>We’re here to support you.

With the current doc guidelines this might sound something like this

> Deploying the Mattermost Mobile App will take time and iteration.

The same content is delivered but the first leaves the reader with the comfort of companionship where the second might stop the reader from trying it at all.

Would this be a suitable example?

Comment (faase) [reply operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: @jasonblais from our chat yesterday I refined the example:

> While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration.
>
>We are here to [support](link-to-support-section) you.

As opposed to

> Deploying the Mattermost Mobile App will take time and iteration.

**Comment by faase** [comment]
I see this got the `Awaiting Submitter Action` label, what action would that be? The addition of the example or more?

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 6 above)
Comment (wiersgallak) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: @faase I would recommend adding these examples to the content in this piece.  Even the TL;DR clarifies the differences well.

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md (hunk 7 above)
Comment (wiersgallak) [reply operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: @faase The refined example makes sense to me as an outside reader.  Recommend incorporating it into the piece.

**Comment by justinegeffen** [comment]
> I see this got the `Awaiting Submitter Action` label, what action would that be? The addition of the example or more?

I added the label as there were some suggestions that seemed to be in flight and needed to be discussed/addressed before the editorial review was complete. :) Also a reminder to me that I hadn't neglected to do a review. :)

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 6 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: Incorporating this I'd add to the **Documentation Style Guide** section

> When writing educational content the audience is placed at the center of attention, opposing technical documentation where it is the documents content.

and change the **Abstract** section to

> An exception is Customer Education content that has no need to outline the content but set more specific expectations for a well-defined audience. 
>
> Clearly name the audience as it can be narrow (System Administrators), but also large (System Administrators, Project Managers, and IT Leaders in general). When setting the expectations answer these questions
>
> - How will the reader benefits from reading the guide - what exact knowledge is taught?
> - How long will it take to work through the guide?
> - What similar topics are not covered?

This should cover all the content in this comment but integrate it seemless into the existing structure. WDYT @wiersgallak

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 8]
```diff
@@ -23,38 +19,37 @@ The title appears in the left-hand navigation and at the top of the table of con
...
-
 ### Abstract
 
-The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. 
-It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the 
-description is displayed in the search results along with the page title. People read the description to 
-help them decide if the document is the one that they want.
+The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the description is displayed in the search results along with the page title. People read the description to help them decide if the document is the one that they want. 
+
+An exception is Customer Education content that has no need to outline the content but set more specific expectations for a well-defined audience.
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]:
```suggestion
An exception is Customer Education content that has no need to outline the content but sets more specific expectations for a well-defined audience. 
```

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 9]
```diff
@@ -23,38 +19,37 @@ The title appears in the left-hand navigation and at the top of the table of con
...
 
-
 ### Abstract
 
-The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. 
-It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the 
-description is displayed in the search results along with the page title. People read the description to 
-help them decide if the document is the one that they want.
+The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the description is displayed in the search results along with the page title. People read the description to help them decide if the document is the one that they want. 
+
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md:31]: I wonder if we can make the following section a bolded subsection: **Customer Education**, so it's easier to see the transition? 3/5 for easier reading.

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 10]
```diff
@@ -23,38 +19,37 @@ The title appears in the left-hand navigation and at the top of the table of con
...
+An exception is Customer Education content that has no need to outline the content but set more specific expectations for a well-defined audience. 
+
+Clearly name the audience as it can be narrow (System Administrators), but also large (System Administrators, Project Managers, and IT Leaders in general). When setting the expectations answer these questions
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]:
```suggestion
Clearly name the audience as it can be narrow (System Administrators), but also large (System Administrators, Project Managers, and IT leaders in general). When setting the expectations answer these questions:
```

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 11]
```diff
@@ -23,38 +19,37 @@ The title appears in the left-hand navigation and at the top of the table of con
...
+
+- How will the reader benefits from reading the guide - what exact knowledge is taught?
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]:
```suggestion
- How will the reader benefit from the guide - what exact knowledge is taught?
```

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 12]
```diff
@@ -23,38 +19,37 @@ The title appears in the left-hand navigation and at the top of the table of con
...
+- How long will it take to work through the guide?
+- What similar topics are not covered?
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: Does this mean those topics are covered in the guide or should be mentioned as not covered in this specific guide?

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md [hunk 13]
```diff
@@ -77,6 +77,18 @@ Use the second person and avoid the first person.
...
+
+For example: 
+> While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration. 
+>
+>We are here to [support](https://link-to-support-section) you.
+
+as opposed to
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]:
```suggestion
As opposed to:
```

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md [hunk 14]
```diff
@@ -77,6 +77,18 @@ Use the second person and avoid the first person.
...
+> Deploying the Mattermost Mobile App will take time and iteration.
+
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: Some extra line spaces here - possibly best to remove them?

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md [hunk 15]
```diff
@@ -77,6 +77,18 @@ Use the second person and avoid the first person.
...
+
+
 **Voice**
 
 Use active voice in preference to passive voice. Active voice has the subject of a sentence doing the action. In passive voice, the subject has an action done to it.
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: Is this accurate for Customer Education content?

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 16]
```diff
@@ -1,18 +1,14 @@
 # Documentation Style Guide
 
-When writing for Mattermost, always think about the audience that will be reading your writing. Mattermost’s 
-audience can vary, but we typically communicate with a technical audience and want to present Mattermost as 
-being a friend of the practitioner in development, IT, and operations. It must be accurate and clear, be 
-presented with a style and tone that is appropriate for technical content, and easy to navigate.
+When writing for Mattermost, always think about the audience that will be reading your writing. Mattermost’s audience can vary, but we typically communicate with a technical audience and want to present Mattermost as being a friend of the practitioner in development, IT, and operations. It must be accurate and clear, be presented with a style and tone that is appropriate for technical or educational content, and easy to navigate.
+
+When writing educational content the audience is placed at the center of attention, opposing technical documentation where it is the documents content.
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]:
```suggestion
An exception is Customer Education content where the audience is placed at the center of attention, as opposed to technical documentation where it is the document's content that is the primary focus.
```
Do you have examples of both? As they are two different ideas it might be helpful to a new writer or contributor to have a point of reference.

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 17]
```diff
@@ -1,18 +1,14 @@
...
 ## Document structure
 
-Structure and organization are an important part of a document's ease of use and its understandability. 
-Information should be organized and presented in a logical order, with similar subjects grouped together in the 
-same section. In most cases, a document has a title, an introductory paragraph, and one or more sections.
+Structure and organization are an important part of a document's ease of use and its understandability. Information should be organized and presented in a logical order, with similar subjects grouped together in the same section. In most cases, a document has a title, an introductory paragraph, and one or more sections.
 
-Try to keep only one topic in a page. Shorter topics are easier to reuse in other documents, are 
-easier to write and edit, and are easier to translate.
+Try to keep only one topic in a page, with the exception of educational guides. Shorter topics are easier to reuse in other documents, are easier to write and edit, and are easier to translate.
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]:
```suggestion
Try to keep only one topic in a page. Shorter topics are easier to reuse in other documents, are easier to write and edit, and are easier to translate. 

An exception to this is Customer Education content which covers multiple topics per page.
```
Maybe have a link to a single-topic technical doc and then one to a CE doc for reference?

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 12 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: What similar topics are not covered in this guide (the guide you are writing)?

The Idea is to set the expectations as clear as possible. For example writing about plug-ins it might be a good idea to make sure it's not about extensions.

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 9 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md:31]: I've thought about it, but then I thought it would be inconsistent with all the other CE subsection of the other topics, so I decided against it 3/5.

How about changing

> An exception is Customer Education content that [...]

to

> An exception is **Customer Education** content that [...]

?

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 16 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: How about 

> While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration. 
>
>We are here to [support](https://link-to-support-section) you.

As opposed to:
> Deploying the Mattermost Mobile App will take time and iteration.

We could include or refer to it in the Voice section, WDYT @justinegeffen

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 17 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: @justinegeffen  I'd like to keep references to a minimum for maintainability purposes and I don't think examples are needed, as all CE content (right now there is only one guide, the Mobile App Deployment Guide) covers multiple topics on a page.

**Review by jasonblais — DISMISSED** [review]

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 9 above)
Comment (justinegeffen) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md:31]: That works for me. :)

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 12 above)
Comment (justinegeffen) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: That's a good explanation. One option is to add it to the final point for clarity.

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 16 above)
Comment (justinegeffen) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: Would the above examples be related to this paragraph or should they be added to the voice section? They would fit better in the voice section if they're not already there.

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 12 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: Sure thing! I'll change it from

> What similar topics are not covered?

to 

> What similar topics are not covered in this guide?

Comment (justinegeffen) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: "Are there similar topics not covered in this guide?" could also be an option?

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md [hunk 18]
```diff
@@ -77,6 +77,16 @@ Use the second person and avoid the first person.
 
 For example: "View the status in the Status pane" as opposed to "We’ll view the status in the Status pane".
 
+An exception is **Customer Education** content which can be written in first or second person to keep the audience focused and interested.
+
+For example: 
+> While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration.
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]:
```suggestion
**While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration.** 
```

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md [hunk 19]
```diff
@@ -77,6 +77,16 @@ Use the second person and avoid the first person.
 
 For example: "View the status in the Status pane" as opposed to "We’ll view the status in the Status pane".
 
+An exception is **Customer Education** content which can be written in first or second person to keep the audience focused and interested.
+
+For example: 
+> While Mattermost provides options to make mobile deployment as easy as possible, it will take time and iteration. 
+>
+>We are here to [support](https://link-to-support-section) you.
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]:
```suggestion
**We are here to [support](https://link-to-support-section) you.**
```
Additional suggestion: Add this directly after the preceding sentence (i.e., remove the line space).

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 16 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: this example is part of this pr in the voice section

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md (hunk 15 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: I'm not 100% sure here, but as of now I'd say yes.

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md (hunk 18 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: Why would we not use markdown block quote here?

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md (hunk 19 above)
Comment (faase) [reply operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: It is supposed to stand alone to give emphasis to the statement.

And as above why should we not use markdown block quotes when we are quoting?

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md (hunk 18 above)
Comment (justinegeffen) [reply operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: I'm just keeping it aligned with the other examples in the document. But you're welcome to use Markdown block quotes if that works better. :)

File: operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md (hunk 19 above)
Comment (justinegeffen) [reply operations/operations/publishing/publishing-guidelines/voice-tone-and-writing-style-guidelines.md]: I've provided a comment above. :)

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 20]
```diff
@@ -23,38 +21,37 @@ The title appears in the left-hand navigation and at the top of the table of con
...
-It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the 
-description is displayed in the search results along with the page title. People read the description to 
-help them decide if the document is the one that they want.
+The abstract provides a high-level overview of the content, includes context, and outlines the value or goal to the reader. It often helps to write the abstract first, as it provides a framework for the content that follows. Keep in mind that the description is displayed in the search results along with the page title. People read the description to help them decide if the document is the one that they want. 
+
+An exception is **Customer Education** content that has no need to outline the content but sets more specific expectations for a well-defined audience. 
+
+Clearly name the audience as it can be narrow (System Administrators), but also large (System Administrators, Project Managers, and IT leaders in general). When setting the expectations answer these questions
```
Comment (jasonblais) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]:
```suggestion
For **Customer Education**, clearly name the audience as it can be narrow (System Administrators), but also large (System Administrators, Project Managers, and IT leaders in general). When setting the expectations answer these questions:
```

**Review by jasonblais — DISMISSED** [review]
One non-blocking suggestion to clarify the example for setting expectations and audience is specific to customer education.

**Review by justinegeffen — DISMISSED** [review]

**Review by jasonblais — DISMISSED** [review]

**Review by justinegeffen — DISMISSED** [review]
I'm happy with this iteration. My only comment is to standardize the examples using markdown formatting as discussed. It's not a blocker but does create consistency and should be quick to implement.

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 6 above)
Comment (wiersgallak) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: @faase looks good to me.

**Review by wiersgallak — DISMISSED** [review]

**Review by justinegeffen — DISMISSED** [review]

**Comment by jasonblais** [comment]
@justinegeffen @faase can you help resolve conflicts and then this should be ready to merge? I was taking a look but  I wasn't 100% sure on what the correct resolution should be

**Comment by justinegeffen** [comment]
Thanks, @jasonblais. The conflict is that the section edited has been moved to the style guide. I've removed these updates and will open a new PR for incorporating them into the style guide.

**Review by justinegeffen — APPROVED** [review]
